edward/inferences/inference.py

```diff
@@ -258,3 +284,48 @@ def finalize(self):
     """
     if self.logging:
       self.train_writer.close()
+
+  def set_log_variables(self, logvars=None, log_max_scalers_per_var=None):
+    """Logs variables to TensorBoard
+
+     For each variable in logvars, creates 'scalar' and / or 'histogram' by
+     calling `tf.summary.scalar` or `tf.summary.histogram`
```

Comment:
To format code, wrap double single ticks, i.e., `\``tf.summary.scalar``\` (without the `\`). Also needs a period at end of sentence.